### PR TITLE
docs: add return_to query parameter to OAS Verification Flow for Native Apps

### PR DESCRIPTION
**Pull request by David-Wobrock**
The idea would be to re-generate the Kratos SDK, so that one can specify a `return_to` URL when creating a Native Verification Flow using the SDK.

## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [ ] I am following the
      [contributing code guidelines](../blob/master/CONTRIBUTING.md#contributing-code).
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got the approval (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added or changed [the documentation](https://github.com/ory/docs).

### CODE REVIEW

File: selfservice/flow/verification/handler.go [hunk 1]
```diff
@@ -123,6 +123,19 @@ func (h *Handler) NewVerificationFlow(w http.ResponseWriter, r *http.Request, ft
...
+// Create Verification Flow Parameters for Native Apps
+//
+// swagger:parameters createNativeVerificationFlow
+//
+//nolint:deadcode,unused
+//lint:ignore U1000 Used to generate Swagger and OpenAPI definitions
+type createNativeVerificationFlow struct {
+	// The URL to return after the flow was completed.
+	//
+	// in: query
```
Comment (aeneasr) [inline selfservice/flow/verification/handler.go:135]: Does this actually work? Native flows don’t use redirects afaik?

**Review by aeneasr — CHANGES_REQUESTED** [review]
I checked and the return_to parameter is not being used anywhere in the verification flow, so this change has no effect. We would instead need to implement return_to support.

What's your use case?

File: selfservice/flow/verification/handler.go (hunk 1 above)
Comment (David-Wobrock) [reply selfservice/flow/verification/handler.go:135]: Not redirections _per se_, but this piece of information is contained in the Verification Flow.

For instance calling `http://127.0.0.1:4433/self-service/verification/api?return_to=http://127.0.0.1:4433/ok` would returns this flow content:
```
{
    "id": "689e20e6-f66c-4bcb-996c-d1f3c87e7670",
    "type": "api",
    "expires_at": "2024-09-19T14:52:43.434168067Z",
    "issued_at": "2024-09-19T13:52:43.434168067Z",
    "request_url": "http://127.0.0.1:4433/self-service/verification/api?return_to=http://127.0.0.1:4433/ok",
    "return_to": "http://127.0.0.1:4433/ok",
    "active": "code",
[...]
```

With a `return_to` key, which we can be used in our logic to know the redirection URL set at the beginning of the flow :)

Comment (David-Wobrock) [reply selfservice/flow/verification/handler.go:136]: I rephrased the sentence.
Let me know what you think :)

**Review by aeneasr — APPROVED** [review]

**Pull request merged**